test/integration_test/common_test.go

```diff
@@ -122,10 +122,8 @@ func setup() error {
 	return nil
 }
 
-func (tc *TestCase) PopulateStorageCluster(cluster *corev1.StorageCluster) error {
+func PopulatePureStorageCluster(tc *TestCase, cluster *corev1.StorageCluster) error {
```

Comment:
is this function still needed except the name need to be populated?

Reply:
It needs to populate the cluster name, and it has one extra function of searching for node selectors like "node0" and replacing them with the `n`th node name in the cluster (basically allowing us to dynamically test node selectors). That's the for loop further down.

Reply:
Do we need to pass the testcase to this function? I don't see it getting used.